php/libraries/Evaluator.class.inc

```diff
@@ -0,0 +1,92 @@
+<?php
+
+    namespace LorisScript;
```

Comment:
Namespaces should be prefixed with `LORIS\` to make it easier to upgrade to a PSR4 autoloader in the future (so: `LORIS\LorisScript`)